Resources/Prototypes/_LostParadise/Entities/Clothing/OuterClothing/softsuits.yml

```
@@ -0,0 +1,261 @@
+      - state: equipped-unshaded
+        shader: unshaded
+  - type: ToggleableClothing
+    clothingPrototype: ClothingHeadEVAHelmetCargo
+
+- type: entity
+  parent: NFClothingOuterEVASuitBase
+  id: ClothingOuterEVASuitSalvage
+  name: salvage specialist EVA suit
+  description: An EVA suit with a built-in helmet commonly issued to salvage specialists.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)